controllers/UsersController.js

```diff
@@ -83,4 +83,77 @@ export default class UsersController {
       }
     }).catch(next);
   }
+
+  /**
+   * Returns a user's details for their profile
+   * @param {*} req - Request object
+   * @param {*} res - Response object
+  * @param {*} next - Incase of errors
```

Comment:
I feel you should reformat your JSDOC comments.

Reply:
I've fixed the header of my commit message but I'll need more information about the JSDoc comments. Thanks

Reply:
Done